unstructured/partition/auto.py

```diff
@@ -407,8 +412,9 @@ def file_and_type_from_url(
     content_type: Optional[str] = None,
     headers: Dict[str, str] = {},
     ssl_verify: bool = True,
+    request_timeout: Optional[int] = None,
 ) -> Tuple[io.BytesIO, Optional[FileType]]:
-    response = requests.get(url, headers=headers, verify=ssl_verify)
+    response = requests.get(url, headers=headers, verify=ssl_verify, timeout=request_timeout)
```

Comment:
can we add a test (maybe using mocker spy) to confirm the timeout is pass down as intended. This is more for preventing future code changes from accidentally break this behavior.

Reply:
@Coniferish do you mind taking a look at a small unit test here? Not a huge priority

Reply:
Already handled in #2013.